utils/spdx/src/test/kotlin/UtilsTest.kt

```diff
@@ -142,6 +142,13 @@ class UtilsTest : WordSpec() {
                 text should endWith("limitations under the License.")
             }
 
+            "return the full license text for a valid SPDX license id with the or later flag set" {
```

Comment:
Nits:
- Maybe single-quote `'or later'`?
- SPDX itself calls `+` an "operator", not a "flag", so we should use the same wording here.